golos.publication/golos.publication.cpp

```diff
@@ -49,6 +49,9 @@ extern "C" {
             execute_action(&publication::set_params);
         if (NN(reblog) == action)
             execute_action(&publication::reblog);
+        if (NN(setcurprcnt) == action)
```

Comment:
Author can't change curator percent after publication. So remove this action.

Reply:
@s-medvedev, why not? current Golos implementation allows such change before first upvote. and looks like there is no issues to give author ability to *increase* percent anytime

Reply:
Ok, you are right.